packages/embark/src/cmd/dashboard/repl.js

```diff
@@ -18,9 +19,9 @@ class REPL {
     }
   }
 
-  enhancedEval(cmd, context, filename, callback) {
-    this.events.request('console:executeCmd', cmd.trim(), function (err, message) {
-      callback(err, message === undefined ? '' : message); // This way, we don't print undefined
+  enhancedEval(cmd, _context, _filename, _callback) {
```

Comment:
If we're never using them, could we simply drop the `_` arguments from the method definition?

Reply:
I prefer keeping those, as they kind of act like documentation.

Reply:
I think the same as André

Reply:
👍